gnocchi/cli/metricd.py

```diff
@@ -187,6 +187,11 @@ def _get_sacks_to_process(self):
                     i for i in six.moves.range(self.incoming.NUM_SACKS)
                     if self.partitioner.belongs_to_self(
                         i, replicas=self.conf.metricd.processing_replicas)]
+        except tooz.NotImplemented:
```

Comment:
you going to move the run_watchers command elsewhere? do we still need this if you do?